io/module_write_netcdf.F90

```diff
@@ -784,8 +785,14 @@ subroutine get_global_attr(fldbundle, ncid, rc)
 
       if (typekind==ESMF_TYPEKIND_I4) then
          call ESMF_AttributeGet(fldbundle, convention="NetCDF", purpose="FV3", &
-                                name=trim(attName), value=varival, rc=rc); ESMF_ERR_RETURN(rc)
-         ncerr = nf90_put_att(ncid, NF90_GLOBAL, trim(attName), varival); NC_ERR_STOP(ncerr)
+                                name=trim(attname), value=varival_i4, rc=rc); ESMF_ERR_RETURN(rc)
+         ncerr = nf90_put_att(ncid, nf90_global, trim(attname), varival_i4); NC_ERR_STOP(ncerr)
+
+      else if (typekind==ESMF_TYPEKIND_I8) then
+         call ESMF_AttributeGet(fldbundle, convention="NetCDF", purpose="FV3", &
+                                name=trim(attname), value=varival_i8, rc=rc); ESMF_ERR_RETURN(rc)
+         varival_i4 = varival_i8
```

Comment:
Why do we need to change the variable from i8 to i4?

Reply:
Without converting the attribute from i8 to i4 I get this runtime error:
```
144:  file: module_write_netcdf.F90 line:          794
144:  NetCDF: Attempting netcdf-4 operation on strict nc3 netcdf-4 file
144: Abort(1) on node 144 (rank 144 in comm 496): application called MPI_Abort(comm=0x84000002, 1) - process
```

Reply:
8-byte integer attributes can now be used after I removed NF90_CLASSIC_MODEL. See https://github.com/DusanJovic-NOAA/fv3atm/commit/7525fa54da7ef29060bf60af67e0aa9209bd0155